libs/pbd/pbd/abstract_ui.h

```diff
@@ -73,20 +73,15 @@ class ABSTRACT_UI_API AbstractUI : public BaseUI
 	};
 	typedef typename RequestBuffer::rw_vector RequestBufferVector;
 
-#if defined(COMPILER_MINGW) && defined(PTW32_VERSION)
```

Comment:
This fails to compile with gcc/clang on macOS/Linux, as well as with our mingw cross compile setup.

I also expect this is still needed if you uses MSYS2/mingw if you are using ptw32 there.